client/src/containers/HomepageContainer/index.js

```diff
@@ -77,20 +97,31 @@ class GridListHome extends React.Component {
     return (
       <div>
         <div style={styles.header}>
-          <p>------------</p>
+          <h1>Better Odds of Football</h1>
+        </div>
+        <div style={styles.headerSubText}>
+          <p>Are you ready to receive the knowing</p>
+        </div>
+        <div style={styles.headerSubText}>
+          <p>----------</p>
+        </div>
+        <div style={styles.introText}>
+          <p>See some of our latest rants below</p>
         </div>
         <div style={styles.gridContainer}>
           <GridList
-            cellHeight={180}
-            padding={15}
+            cols={2}
+            cellHeight={350}
+            padding={25}
             style={styles.gridList}
           >
             {this.state.articles.map(({ fields, sys }, index) => {
               return (
                 <GridTile
                   key={sys.id}
                   title={fields.articleTitle}
-                  onClick={this.viewPost}
+                  containerElement={<FlatButton
```

Comment:
I changed this in a later commit :)
It'll use Link rather than FlatButton. Cleaner that way I think.